internal/server/server.go

```diff
@@ -266,10 +266,14 @@ func (s *Server) listen() error {
 }
 
 func (s *Server) setupServer(server *http.Server) (net.Addr, error) {
+	if server.Addr == "" {
+		server.Addr = "127.0.0.1:"
```

Comment:
Is the trailing semi-colon for the port intentional?

Reply:
I don't think this does what you expect @ssoroka. `server.Addr` should never be empty since there are defaults, e.g. `:8080` for the HTTP server, `:8443` for the HTTPS server